src/system-core.js

```diff
@@ -270,9 +271,9 @@ function postOrderExec (loader, load, seen) {
     }
   });
   if (depLoadPromises)
-    return Promise.all(depLoadPromises).then(doExec);
+    return load.E = Promise.all(depLoadPromises).then(doExec);
```

Comment:
Previously `load.E` the module execution promise. I think it should be `depLoadPromises` then execution promise.